sonic_installer/main.py

```diff
@@ -763,6 +763,7 @@ def cleanup():
     "radv",
     "restapi",
     "sflow",
+    "stp",
```

Comment:
>stp [](http://example.com/codeflow?start=5&length=3)

If deployed do not use stp feature at all, will it increase image size? will it break in runtime?

Reply:
If STP is not enabled , it will not break at run time.